src/Symfony/Component/DependencyInjection/Tests/Compiler/AutowireRequiredMethodsPassTest.php

```diff
@@ -70,8 +73,13 @@ public function testSetterInjectionWithAttribute()
         $this->assertSame([['setFoo', []]], $methodCalls);
     }
 
+    /**
+     * @group legacy
+     */
     public function testExplicitMethodInjection()
     {
+        $this->expectDeprecation('Since symfony/dependency-injection 6.3: The "@required" annotation will be removed. Use the attribute "Symfony\Contracts\Service\Attribute\Required" instead.');
```

Comment:
```suggestion
        $this->expectDeprecation('Since symfony/dependency-injection 6.3: The "@required" annotation will be removed. Use the "Symfony\Contracts\Service\Attribute\Required" attribute instead.');
```